.eslintrc.json

```diff
@@ -1,7 +1,6 @@
 {
   "extends": [
     "eslint:recommended",
-    "plugin:react-hooks/recommended",
```

Comment:
removed this because `next/core-web-vital` includes it by default and my command line was lecturing me on it